pod/annotations.go

```diff
@@ -502,6 +502,9 @@ func parseAnnotations(pod *corev1.Pod, pConf *Config) error {
 		err = multierror.Append(err, sErr)
 	}
 
+	if err == nil {
+		return nil
+	}
 	return err.ErrorOrNil()
```

Comment:
Doesn't `ErrorOrNil` do this?

Why can't it just `return err` anyway?

Reply:
I have fixed the lint violation only.